InterruptClient.rb

```diff
@@ -0,0 +1,231 @@
+require 'socket'
+require 'json'
+
+require_relative './Console'
+
+class InterruptClient
+
+	CMD_QUIT = "\\"
+	START_MSG = 'Welcome! What is your name?'
+	INSTRUCTIONS = "Start typing to join the chat! To quit anytime, type #{CMD_QUIT}"
+	FAREWELL = "\r\nbye"
+	PROMPT = '> '
+
+	MAX_MSG_LENGTH = 3000 # max length of incoming message read
```

Comment:
This is a bit of a tangent, but datagrams of this size will almost certainly be fragmented into multiple packets and require reassembly if sent across the Internet. This should be handled transparently for you, but wanted to point it out if you weren't aware.

Further reading: https://en.wikipedia.org/wiki/Maximum_transmission_unit#MTUs_for_common_media